index.js

```diff
@@ -24,24 +39,25 @@ module.exports = function api(options) {
     }
 
     //options: website, username, password, basic (hashed)
-    var website = options.website,
-        headers = {};
+    var website = options.website;
+    var headers = {};
 
-    if(options.username && options.password){
-        headers.Authorization =  "Basic " +
+    if (options.username && options.password) {
+        headers.Authorization = "Basic " +
             new Buffer(options.username + ":" + options.password)
-            .toString("base64");
+                .toString("base64");
     }
 
-    if(options.basic){
-      headers.Authorization =  "Basic " + options.basic;
+    if (options.basic) {
+        headers.Authorization = "Basic " + options.basic;
     }
 
     var r = request.defaults({
         baseUrl: "https://" + website + ".scm.azurewebsites.net/",
         headers: headers,
     });
-    return {
+
+    return promisifyApi({
```

Comment:
This will mutate each module, but is non-destructive to the existing methods. If there is a method called `list` then it will have a new sibling added called `listAsync`.